### PR TITLE
Fixes evidences callbacks in fa application.

### PR DESCRIPTION
**Pull request by ZASMan**
ME-180637092

### CODE REVIEW

**Review by markgoho — DISMISSED** [review]
need a ticket reference

File: components/financial_assistance/app/models/financial_assistance/application.rb [hunk 1]
```diff
@@ -1568,22 +1568,20 @@ def create_evidences
...
     def build_evidences(types, applicant)
-      evidences = types.collect do |type|
-        key, title = type
-
-        next if applicant.evidences.by_name(key).present?
-        case key
+      evidences = []
+      types.each do |type|
+        next if applicant.evidences.by_name(type).present?
+        case type
```
Comment (harshaellanki) [inline components/financial_assistance/app/models/financial_assistance/application.rb:1574]: here type is not symbol, it is an array

Comment (ZASMan) [reply components/financial_assistance/app/models/financial_assistance/application.rb:1576]: Good point let me fix that

**Review by markgoho — DISMISSED** [review]
pull brakeman changes out into a separate PR

**Review by bill-transue — APPROVED** [review]
This will fix the evidences callback so that nil evidences are no longer created.

**Comment by ZASMan** [comment]
> pull brakeman changes out into a separate PR

They're pulled out

**Pull request merged**